test/unit/component/parser/json/BpmnJsonParser.lane.test.ts

```diff
@@ -115,6 +115,41 @@ describe('parse bpmn as json for lane', () => {
     });
   });
 
+  it('json containing one process with a single lane with flowNodeRef as object & flowNode not parsed', () => {
```

Comment:
:question: why did you add this test? was it missing and you add it now even if not related to `end event`?

Reply:
This test is existing before. I just rebase the branch, but I don't know why it doesn't exist in the master branch.
Maybe a bad old rebase ?

Reply:
ok, fair enough :smile_cat: